src/controls/user.js

```diff
@@ -167,13 +158,7 @@ router.post("/register", async (req, res, next) => {
     })(req, res, next);
 });
 
-// router.post("/createuser", async (req, res, next) => {
-//     passport.authenticate("signup", async (err, user, info) => {
-//         if (err) return res.status(500).send({ message: err });
-//         if (!user || info) return res.status(300).send({ message: info });
-//         res.status(200).send(user);
-//     })(req, res, next);
-// });
+let refreshTokens = [];
```

Comment:
can we move this array to some safer place ?? Why aren't we keeping it in Mongo or redis ?

Reply:
We should move but due to time constraint we will do that once the tasks week1 tasks are completed.